src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -4820,8 +4820,9 @@ export interface GoAnalysis {
    *
    * @param boardState - The initial board state to use for the new game, in the format used by getBoardState().
    * @param komi - Optional komi value to set for the game. Defaults to 5.5.
+   * @param nextPlayerIsWhite - Optional. Whether or not the next player to play is the white player. Defaults to true.
```

Comment:
Boolean optional params typically should default to false, and in Go typically black goes first?